lib/api/apiUtils/quotas/quotaUtils.js

```diff
@@ -147,8 +147,9 @@ function _evaluateQuotas(
                     if (err || inflight < 0) {
                         return parallelDone(err);
                     }
+                    // Metrics are served as BigInt strings
                     if (!isMetricStale(accountMetrics, 'account', account.account, action, inflight, log) &&
-                        accountMetrics.bytesTotal + inflightForCheck > accountQuota) {
+                        BigInt(accountMetrics.bytesTotal || 0) + BigInt(inflightForCheck || 0) > accountQuota) {
```

Comment:
shoud these conversions not be done by `VaultClient` ?

Reply:
I need to double check this one: vaultclient uses a unique function to handle response, hence we cannot easily do it. Plus it is also used as a cli tool so we need to ensure this breaks nothing.